frontend/src/metabase/components/AddToDashSelectDashModal.jsx

```diff
@@ -58,10 +60,21 @@ export default class AddToDashSelectDashModal extends Component {
                     title="Add Question to Dashboard"
                     closeFn={this.props.closeFn}
                 >
+                <div className="flex flex-column">
```

Comment:
Hrm yeah that's not exactly what we want is it :P Let me pull your branch and try something and get back to you shortly.